owlplug-client/src/main/java/com/owlplug/core/controllers/OptionsController.java

```diff
@@ -200,12 +218,16 @@ public void refreshView() {
     storeSubDirectoryCheckBox.setSelected(this.getPreferences().getBoolean(ApplicationDefaults.STORE_SUBDIRECTORY_ENABLED, true));
     warningSubDirectory.setVisible(!this.getPreferences().getBoolean(ApplicationDefaults.STORE_SUBDIRECTORY_ENABLED, true));
     storeDirectoryCheckBox.setSelected(this.getPreferences().getBoolean(ApplicationDefaults.STORE_DIRECTORY_ENABLED_KEY, false));
+    storeByMakerCheckBox.setSelected(this.getPreferences().getBoolean(ApplicationDefaults.STORE_BY_MAKER_ENABLED_KEY, false));
     storeDirectoryTextField.setText(this.getPreferences().get(ApplicationDefaults.STORE_DIRECTORY_KEY, ""));
 
     if (!storeDirectoryCheckBox.isSelected()) {
       storeDirectoryTextField.setDisable(true);
+      storeDirectoryTextField.setVisible(false);
```

Comment:
there is a modification here, so that the TextField 'hides' when false, and the other checkbox is initialized in the same way

Reply:
LGTM